src/DurableSDK/OrchestrationInvoker.cs

```diff
@@ -11,58 +11,98 @@ namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
     using System.Linq;
     using System.Management.Automation;
 
-    using PowerShellWorker.Utility;
     using Microsoft.Azure.Functions.PowerShellWorker.Durable.Actions;
 
     internal class OrchestrationInvoker : IOrchestrationInvoker
     {
-        public Hashtable Invoke(OrchestrationBindingInfo orchestrationBindingInfo, IPowerShellServices pwsh)
+        private IExternalOrchestrationInvoker externalInvoker;
+        internal static string isOrchestrationFailureKey = "IsOrchestrationFailure";
+
+        public Hashtable Invoke(
+            OrchestrationBindingInfo orchestrationBindingInfo,
+            IPowerShellServices powerShellServices)
         {
             try
             {
-                var outputBuffer = new PSDataCollection<object>();
-                var context = orchestrationBindingInfo.Context;
+                if (powerShellServices.HasExternalDurableSDK())
```

Comment:
Unfortunately, the diff in this section of the code makes the edit seem more complicated than it is.
In short, the **previous** logic of the `Invoke` method in here is now contained within the `InvokeInternalDurableSDK` . However, for the rest of this file, the diff will show parts of the old method interleaved in weird ways with new methods.